server/app.js

```diff
@@ -1,3 +1,5 @@
+require("dotenv").config();
```

Comment:
This is already done in `bin/www` if you run the server by `npm run dev` the env vars will be added to the process. It only needs to happen once for the process to have reference to all of them.